src/patternfly/components/Menu/menu.scss

```
@@ -44,6 +46,9 @@
   // * Menu list item
   --#{$menu}__list-item--Color: var(--pf-t--global--text--color--regular);
   --#{$menu}__list-item--BackgroundColor: var(--pf-t--global--background--color--action--plain--default);
+  --#{$menu}__list-item--BorderWidth: var(--pf-t--global--high-contrast--border--width--control--default);
+  --#{$menu}__list-item--BorderColor: transparent;
+  --#{$menu}__list-item--hover--BorderColor: var(--pf-t--global--border--color--high-contrast);
```

Should we update this to set the color by default and use the action--plain tokens?

```suggestion
  --#{$menu}__list-item--BorderWidth: var(--pf-t--global--border--width--action--plain--default);
  --#{$menu}__list-item--BorderColor: var(--pf-t--global--border--color--high-contrast);
  --#{$menu}__list-item--hover--BorderWidth: var(--pf-t--global--border--width--action--plain--hover);
```